extensions/default/src/Panels/PanelStudyBrowser.tsx

```diff
@@ -17,18 +17,19 @@ function PanelStudyBrowser({
   requestDisplaySetCreationForStudy,
   dataSource,
 }) {
-  const { hangingProtocolService, displaySetService, uiNotificationService } =
+  const { hangingProtocolService, displaySetService, uiNotificationService, GoogleSheetsService } =
```

Comment:
we seem to use panelstudybrowser and panelstudybrowsertracking in extensions.

Which one are we going to move forward with? We should probably keep these in the extensions/modes if they are already being managed in that side.